docs/guidelines-general.md

```diff
@@ -0,0 +1,58 @@
+# General Guidelines
+
+This document contains general guidelines and useful references related to the review process.
+
+We're still in the early stages of the process, trying to figure out how this should (could) work.
+This is a live document and we should expect it to be updated as time goes on.
+
+## What is a code review?
+
+TBC
+
+## What is a cross-project code review?
+
+TBC
+
```

Comment:
I felt like a little justification was needed about the whole thing, so I've made it as a suggestion here. Happy to PR this separately if you feel this is encroaching on the remit of this PR.

```suggestion
## Why does this exist?

Due to the nature of REG, we don't always get to directly help each other when writing code. This is an idea to make that a little better: giving an opportunity for your colleagues to look through code you've written, and suggest changes where required. As well as giving us all practice being part of code review, you get to show everyone the awesome work you've been doing, which we all want to see!

## Who is this for?

People in REG that want to
- practice going through the process of code review, both as a reviewee and reviewer
- get suggestions on their work when they are unsure how to proceed
  - at the same time, we will all learn a lot about new frameworks, languages and ideas we've never seen in our main projects!
- build a catalogue of code examples we can look back on when thinking about best practices

```

Reply:
Great; love it! Thank you @phinate!